### ISSUE TITLE
CMake install, win32 library name

### ISSUE THREAD

**Issue by aleksey-nikolaev** [alanxz/rabbitmq-c#354]
In rabbitmq-c\librabbitmq\CMakeLists.txt (154) the library name concatenated with RMQ_SOVERSION this is the reason that the [FindRabbitmqc.cmake](https://github.com/alanxz/SimpleAmqpClient/blob/master/Modules/FindRabbitmqc.cmake):
```
FIND_LIBRARY(Rabbitmqc_LIBRARY
     NAMES rabbitmq
     HINTS ${Rabbitmqc_DIR}/lib
)
```
can`t find the rabbitmq.4.lib

If library name must contain version or some extended suffix/prefix text than install directory must have RabbitMQConfigure.cmake, like how OpenCV does for it`s custom named libraries, for example.

**Comment by alanxz** [comment]
Did you intend to file this in the [alanxz/SimpleAmqpClient](https://github.com/alanxz/SimpleAmqpClient) repository?

**Comment by aleksey-nikolaev** [comment]
Yes, I have in mind FindRabbitmqc.cmake in  [alanxz/SimpleAmqpClient](https://github.com/alanxz/SimpleAmqpClient) repository. I used copy of FindRabbitmqc.cmake for load settings for an external project. But [find_package](https://cmake.org/cmake/help/v3.5/command/find_package.html) did not find rabbitmq.lib on win32 and I had to hardcode rabbitmq.4.

Most external cmake projects have it`s own _< name >Config.cmake_ files and it is best way, I think, because the developers of those projects can make a file names or path in package contents how they want.
